test/stdlib/MultilineString.swift

```diff
@@ -0,0 +1,124 @@
+// RUN: %target-run-stdlib-swift | %FileCheck %s
```

Comment:
Move to `test/Parser`.  Please change the test not to build and run, but only to invoke the frontend and verify the AST:

```
// RUN: %target-swift-frontend -dump-ast %s 2>&1 | %FileCheck %s

_ = """
    One
    Two
    """
// CHECK: "One\nTwo"
```